Userland/Libraries/LibWeb/Painting/BorderPainting.cpp

```diff
@@ -24,10 +24,12 @@ BorderRadiusData normalized_border_radius_data(Layout::Node const& node, Gfx::Fl
 
     // Scale overlapping curves according to https://www.w3.org/TR/css-backgrounds-3/#corner-overlap
     auto f = 1.0f;
-    f = min(f, rect.width() / (float)(top_left_radius_px + top_right_radius_px));
-    f = min(f, rect.height() / (float)(top_right_radius_px + bottom_right_radius_px));
-    f = min(f, rect.width() / (float)(bottom_left_radius_px + bottom_right_radius_px));
-    f = min(f, rect.height() / (float)(top_left_radius_px + bottom_left_radius_px));
+    auto width_reciprocal = 1.0f / rect.width();
```

Comment:
lhs computes `rect.width / something`, rhs computes `something / rect.width`, no? Or am I missing something?

Did you mean to use `/ f` instead of `* f` now further down? Or do `f = 1/f` after the mins?

Reply:
You are completely correct, thank you